jina/helper.py

```diff
@@ -92,12 +98,28 @@ def call_obj_fn(obj, fn: str):
 
 
 def touch_dir(base_dir: str) -> None:
+    """
+    Create a directory from given path if it doesn't exist.
+
+    :param base_dir: Path of target path.
+    :returns: None
```

Comment:
Just out of curiosity: If there is no return value, should this still be present?

Reply:
Removed